packages/headless/esbuild.mjs

```diff
@@ -149,7 +149,7 @@ async function buildBrowserConfig(options, outDir) {
     plugins: [
       alias({
         'coveo.analytics': resolveEsm('coveo.analytics'),
-        'cross-fetch': resolve('.', 'fetch-ponyfill.js'),
+        '@coveo/please-give-me-fetch': resolve('.', 'fetch-ponyfill.js'),
```

Comment:
Replaced cross-fetch with a simpler ponyfill for Node (browser unimpacted)

Reply:
It's. a "fake" packages. All the logic about it is in the esbuild. Essentially you can see it as a branching point which point to either fetch-ponyfill or fetch-ponyfill-node.
It'll be removed with v3